shared/test/integration/helpers/view.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    getRootElement: function (rootSelector, method) {
+        return this.client.findElement(rootSelector, method);
+    },
+    findElement: function (selectorObject) {
+        return this.client
+        .findElement(selectorObject.locator, selectorObject.by);
+            
+    },
+    rootDisplayed: function () {
+        return this.getRootElement().displayed();
+    },
+    waitForDisplay: function (selectorObject) {
+    	var selector = selectorObject;
+        var self = this;
+        return this.client.waitFor(function () {
+            return self.findElement(selector).displayed();
+        });
+    },
+    waitForHide: function () {
```

Comment:
It might be a good idea to expose that method here, however the implementation is already there: https://github.com/mozilla-b2g/gaia/blob/master/tests/jsmarionette/plugins/marionette-helper/index.js#L178